internal/pkg/service/common/etcdop/op/atomic.go

```diff
@@ -265,7 +265,7 @@ func (v *AtomicOp[R]) DoWithoutRetry(ctx context.Context, opts ...Option) *TxnRe
 
 	// Create IF part of the transaction
 	var cmps []etcd.Cmp
-	for _, op := range removeOpsOverlaps(tracker.Operations()) {
+	for _, op := range tracker.Operations() {
```

Comment:
I found bug in the `removeOpsOverlaps` function, see example:

![image](https://github.com/keboola/keboola-as-code/assets/19371734/db593462-5c99-404e-bc45-3eee9e9ad5ba)

![image](https://github.com/keboola/keboola-as-code/assets/19371734/0b39b99f-1792-4d3f-bc31-a80ef28cd2a2)

-------------------

Function purpose was to remove overlaps in tracked operations from Read Phase.

For example:
- Range `GET [aaa/, xxx0)` contains range  `GET [bbb/, ccc0)`, so `[bbb/, ccc0)` can be ignored, form Write Phase conditions.

In the previous iteration of the Stream API it was used, but now it is not used at all (see etcd snapshots in the next PR).

So I decided to remove this feature at all. It is complex and there is no reason to maintain it.

----------------------

B